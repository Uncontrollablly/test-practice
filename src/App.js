import "@/App.css";
import Hello from "@/components/hello/Hello";
import User from "@/components/user/User";
import Card from "./components/card/Card";
import Contact from "./components/contact/Contact";
import Toggle from "./components/toggle/Toggle";

function App() {
  return (
    <div className="App">
      <Hello></Hello>
      <User></User>
      <Toggle></Toggle>
      <Contact></Contact>
      <Card></Card>
    </div>
  );
}

export default App;
