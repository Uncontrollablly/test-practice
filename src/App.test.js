import renderer from "react-test-renderer";
import App from "./App";

test("can render components", () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
