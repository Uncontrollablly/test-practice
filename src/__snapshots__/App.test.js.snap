// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render components 1`] = `
<div
  className="App"
>
  <span>
    嘿，陌生人
  </span>
  加载中...
  <button
    data-testid="toggle"
    onClick={[Function]}
  >
    Turn on
  </button>
  <div>
    <address>
      联系 
      ，通过
       
      <a
        data-testid="email"
        href="mailto:undefined"
      >
        email
      </a>
      或者他们的
       
      <a
        data-testid="site"
      >
        网站
      </a>
      。
    </address>
    <div />
    <div>
      Loading...
    </div>
  </div>
  <button
    data-testid={1}
    onClick={[Function]}
  >
    1
  </button>
  <button
    data-testid={2}
    onClick={[Function]}
  >
    2
  </button>
  <button
    data-testid={3}
    onClick={[Function]}
  >
    3
  </button>
  <button
    data-testid={4}
    onClick={[Function]}
  >
    4
  </button>
</div>
`;
